### ISSUE TITLE
Add python-concurrent.futures as dep of Salt on Debian and Ubuntu

### ISSUE THREAD

**Issue by dmurphy18** [saltstack/salt-pack#96]
Need to add python-concurrent.futures as a dependency of salt-common for Debian 7 & 8, Ubuntu 12, 14 & 16 since python-tornado requires it but doesn't list it as a dependency.

2015.8.11
 [ ] Debian 8 
 [ ] Debian 7
 [ ] Ubuntu 16.04
 [ ] Ubuntu 14.04
 [ ] Ubuntu 12.04

2016.3.1
 [ ] Debian 8 
 [ ] Debian 7
 [ ] Ubuntu 16.04
 [ ] Ubuntu 14.04
 [ ] Ubuntu 12.04

**Comment by ahammond** [comment]
This indirectly breaks salt-cloud by breaking salt-bootstrap. @dmurphy18 please oh please oh please!

**Comment by dmurphy18** [comment]
@ahammond Thanks for the heads up. I shall reevaluate and probably adjust python-tornado to include the dependency issue.

**Comment by ahammond** [comment]
@dmurphy18 that'd work. Until then `script_args: -p python-futures stable latest` might unblock me.

**Comment by dmurphy18** [comment]
@ahammond on repo.saltstack.com the package you need is named python-concurrent.futures

**Comment by dmurphy18** [comment]
@ahammond will be a dependency on python-tornado in the next point release

**Comment by ahammond** [comment]
I just added python-futures and apt correctly picked
python-concurrent.futures.

**Comment by dmurphy18** [comment]
Closing this since resolved with recent release of 2016.3.1 with dependency on python-tornado.
If it still has issues, please open another issue since this was to add the dependency.
